src/locales/README.md

```diff
@@ -0,0 +1,24 @@
+# Ory Elements Default Translations
+
+This directory contains the default translations for Ory Elements. These
+translations are also used in the Ory Cloud Account Experience. Missing
+translations automatically fall back to the English version.
+
+## Automation
+
+The English base language file is auto-generated from various sources. Please
+update it by running `npm run generate-locales`.
```

Comment:
Should we make this a cronjob and create a PR with the result if there are changes?

Reply:
Yeah, I was under the impression that there was such an action, but maybe I never came to do it.